docs/concepts/exchange.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 7
+sidebar_label: "Exchange Backend"
+title: Asset Exchange Backend
+description: Using a ledger device to confirm your actions
+---
+
+Near Treasury uses [RHEA Finance](#about-rhea-finance) as the backend provider for all [asset exchange](../assets.md) features available to Treasury users.
+
+In this article you'll find additional information about RHEA and the [Auto Router algorithm](#rhea-auto-router) used to get better prices, and better liquidity utilization compared to regular exchanges.
+
+## About RHEA Finance
+
+[RHEA Finance](https://rhea.finance/) is an automated liquidity protocol implemented on the [NEAR blockchain](https://near.org).
```

Comment:
it's actually a platform, from their doc:
"RHEA Finance is a community-led, multi-purpose DeFi platform built on the NEAR Protocol."